library/public/interactor.h

```diff
@@ -234,24 +223,33 @@ class F3D_EXPORT interactor
   /**
    * Play a VTK interaction file.
    */
-  virtual bool playInteraction(const std::string& file) = 0;
+  virtual bool playInteraction(
+    const std::string& file, double deltaTime = 0.03, std::function<void()> userCallBack = nullptr) = 0;
```

Comment:
I would set the default to 30fps.

```suggestion
    const std::string& file, double deltaTime = 1.0 / 30.0, std::function<void()> userCallBack = nullptr) = 0;
```